### ISSUE TITLE
Newlines in table data 

### ISSUE THREAD

**Issue by torohill** [thephpleague/climate#159]
If there are newlines in the table data these cause the layout to break. For example:

```php
$climate->table([
    ['foo'=>"bar\nbar", 'baz'=>'foobaz'],
    ['foo'=>'bar', 'baz'=>'foobar'],
]);

--------------------
| foo     | baz    |
====================
| bar
bar | foobaz |
--------------------
| bar     | foobar |
--------------------
```

What do you think about adding an option to format newlines so the table looks like this?
```
----------------
| foo | baz    |
================
| bar | foobaz |
| bar |        |
----------------
| bar | foobar |
----------------
```

**Comment by duncan3dc** [comment]
Sounds like a good idea (although rather than an option I'd expect that to just be standard behaviour), not sure how difficult/practical it will be though, will take a look in the next week

**Comment by duncan3dc** [comment]
This is fixed in [3.6.0](https://github.com/thephpleague/climate/releases/tag/3.6.0) thanks for the code!
